README-dev.md

```diff
@@ -21,58 +21,58 @@ This is done automatically by `dvp build`.
 Delphix Engine and wrappers changes only impact the plugin build.
 
-## Development process
+## Development
+ 
+### Development process
 
 At a very high level, our development process usually looks like this:
 
 1. Create a fork of the delphix/virtualization-sdk repository.
 2. Clone the forked repository.
 3. Make changes to SDK code. Test these changes manually and with unit tests. Iterate on this until you have everything working.
-4. Commit your changes and build all Python package distributions. Make sure the version number of the packages is updated appropriately.
-5. Publish Python distributions to artifactory. 
-7. Run blackbox against the newly uploaded SDK version.
-7. Publish a pull request to the delphix/virtualization-sdk once your code is ready for review.
-8. Once the pull request is approved, it will merged into delphix/virtualization-sdk repository.
+4. Bump major/minor/patch/build version depending on the scope of the change.
+5. Commit your changes. Make sure the version number of the packages is updated appropriately.
+6. Push your changes to a branch in the forked repository.
+7. Run blackbox tests against that branch.
+8. Publish a pull request to the delphix/virtualization-sdk once your code is ready for review.
+9. Once the pull request is approved, merge the pull request into delphix/virtualization-sdk repository.
 
 These steps are described in more detail below.
 
-## Local SDK Development
+### Development environment
+Development should be done in a personal virtualenv. To setup the virtual environment:
 
-To setup local development, refer to README-dev.md in the `tools` directory. This walks through the setup of a local virtualenv for development. This should be done for _all_ SDK changes.
+1. `virtualenv /path/to/env/root`. This should be a Python 2.7 virtualenv.
+2. `source ~/path/to/env/root/bin/activate`.
 
-### Configure pip index
+### Installing the SDK from source
+Go into one of the package directories (common, dvp, libs, platform, tools) and run the following commands:
+1. Install the package's development dependencies: `pip install -r requirements.txt`.
+2. Install the package itself (use `-e` flag if you want to install the package in editable mode): `pip install .`.
 
-`dvp build` executes `pip` to install the wrappers. By default `pip` looks at pypi.org for packages to install. Internal builds of the SDK are published to artifactory, not pypi. In order to configure pip to look at artifactory, create a file at `<virtualenv-root>/pip.conf` that contains:
+One of the SDK dependencies - dvp-api - is currently hosted on [test PyPi](https://test.pypi.org/project/dvp-api/). 
+By default `pip` looks at pypi.org for packages to install. In order to successfully install the SDK, you have to 
+configure pip to search an additional package repository - test.pypi.org. To do that, create a file at 
+`<virtualenv-root>/pip.conf` that contains:
```

Comment:
this should be `3.`?
suggestion start with what the developer to do and follow it up with why, something like:

3. To do that, create a file at 
`<virtualenv-root>/pip.conf` that contains .....
......
This is needed because one of the SDK dependencies - dvp-api - is currently hosted on [test PyPi](https://test.pypi.org/project/dvp-api/).  By default `pip` looks at pypi.org for packages to install. In order to successfully install the SDK, you have to configure pip to search an additional package repository - test.pypi.org
```

Reply:
I changed the order of these steps as creating the `pip.conf` is necessary for running `pip install`.

To install the SDK, follow these steps:

1. Create a file at 
    `<virtualenv-root>/pip.conf` that contains:
    
    ```
    [install]
    index-url=https://pypi.org/simple/
    extra-index-url=https://test.pypi.org/simple/
    ```
   
   One of the SDK dependencies - dvp-api - is currently hosted on [test PyPi](https://test.pypi.org/project/dvp-api/). 
   By default `pip` looks at pypi.org for packages to install. In order to successfully install the SDK, you have to 
   configure pip to search an additional package repository - test.pypi.org.
2. Go into one of the package directories (common, dvp, libs, platform, tools) and run the commands below.
3. Install the package's development dependencies: `pip install -r requirements.txt`.
4. Install the package itself (use `-e` flag if you want to install the package in editable mode): `pip install .`.

Reply:
sounds good.